peakina/datasource.py

```diff
@@ -31,6 +31,7 @@ class DataSource:
     uri: str
     type: TypeEnum = None
     match: MatchEnum = None
+    scheme: str = None
```

Comment:
Why adding `scheme` here since it's already in `__post_init__` ? IDE warning?

Reply:
tests were failing https://circleci.com/gh/ToucanToco/peakina/77?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link, it looks like __post_init__ is not allowed to write new attributes :thinking:

Reply:
Yet tests work fine on my machine 🤔 Is it possible only with python 3.7?